wp-content/themes/darklight-site/front-page.php

```diff
@@ -3,15 +3,44 @@
 * Template Name: front page
 */
 get_header();
-
 $image = wp_get_attachment_image_src(get_post_thumbnail_id($post->ID), 'single-post-thumbnail')[0]
 
 ?>
-
+<div class="body">
+<div class= "container">
+  <div class= "row">
+    <div class= "col-6">
 <section class="banner">
```

Comment:
can you out-dent this, it might after other elements down the page so maybe check those out